src/lib.rs

```diff
@@ -36,7 +36,6 @@ extern crate alloc;
 extern crate cpp_demangle;
 #[cfg(feature = "fallible-iterator")]
 pub extern crate fallible_iterator;
```

Comment:
Is there a reason the feature gated crates can't be removed too?

Reply:
These could be removed too, just wasn't handled by `cargo fix`

Reply:
After a second look, the public crates need to be kept. Maybe the removal of gimli below is a cargo fix bug.